lib/src/values/logic_value.dart

```diff
@@ -761,6 +767,20 @@ abstract class LogicValue {
       // ignore: avoid_dynamic_calls
       _doCompare(other, (a, b) => (a >= b) as bool);
 
+  /// Power operation
+  ///
+  /// Both inputs [a] and [b] are either of type [int] or [BigInt].
+  /// Returns [a] raise to the power [b] of same input type.
+  dynamic _powerOperation(dynamic a, dynamic b) {
+    if (a is BigInt) {
+      // ignore: avoid_dynamic_calls
+      return a.pow(b.toInt() as int);
```

Comment:
We should add an explicit exception if the value of `b` as a BigInt won't fit in an `int`.  I think this should work: https://api.flutter.dev/flutter/dart-core/BigInt/isValidInt.html.  Otherwise, we just truncate and silently do the wrong thing.  (even though it's unlikely someone needs more than 64 bits for their exponent intentionally).